wasp/src/main/java/com/orhanobut/wasp/Wasp.java

```diff
@@ -189,6 +189,9 @@ public Wasp build() {
         }
 
         private void init() {
+            if (endPointUrl == null) {
```

Comment:
If client library is throwing an exception then it should have throws name in method signature, otherwise the app can crash because of misconfiguration. Developer should be notified as it can crash the app